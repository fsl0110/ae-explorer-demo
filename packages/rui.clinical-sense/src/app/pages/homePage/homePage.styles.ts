import styled from "styled-components";

// basic exaple for styling html tags
export const HomePageStyles = styled.div`
  padding: 1rem;
`;
