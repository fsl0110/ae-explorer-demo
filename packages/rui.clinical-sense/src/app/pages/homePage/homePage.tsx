import React, {FC} from "react";
import {HomePageStyles} from "./homePage.styles";

interface Props {
  count?: number;
}

export const HomePage: FC<Props> = ({count}) => (
  <div>
    <HomePageStyles>
      <h2>HomePage </h2>
    </HomePageStyles>
  </div>
);
