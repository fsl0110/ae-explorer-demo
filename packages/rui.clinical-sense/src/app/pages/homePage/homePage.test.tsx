import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import {HomePage} from "./homePage";

describe("<HomePage>", () => {
  it("Snapshot without count", () => {
    expect(toJson(shallow(<HomePage />))).toMatchSnapshot();
  });

  it("Snapshot with count", () => {
    expect(toJson(shallow(<HomePage count={1} />))).toMatchSnapshot();
  });
});
