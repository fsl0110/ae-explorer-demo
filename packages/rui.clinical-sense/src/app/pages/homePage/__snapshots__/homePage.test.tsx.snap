// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage> Snapshot with count 1`] = `
<div>
  <styled.div>
    <h2>
      HomePage 
    </h2>
    <p>
      Recieved Count: 
      1
    </p>
    <CounterRedux />
  </styled.div>
</div>
`;

exports[`<HomePage> Snapshot without count 1`] = `
<div>
  <styled.div>
    <h2>
      HomePage 
    </h2>
    <CounterRedux />
  </styled.div>
</div>
`;
